implementations/rust/ockam/ockam_command/src/vault/create.rs

```diff
@@ -10,7 +11,10 @@ use ockam_core::api::Request;
 #[derive(Clone, Debug, Args)]
 pub struct CreateCommand {
     #[command(flatten)]
-    node_opts: NodeOpts,
+    node_opts: Option<NodeOpts>,
+
+    #[arg(short, long)]
+    vault_name: Option<String>,
```

Comment:
You should use the [`group`](https://docs.rs/clap/latest/clap/_derive/_tutorial/index.html#argument-relations) clap attribute here so the user can only pass one of the two arguments.